agnoster.zsh-theme

```diff
@@ -122,10 +122,18 @@ prompt_status() {
 
 # Display current virtual environment
 prompt_virtualenv() {
-  if [[ -n $VIRTUAL_ENV ]]; then
+  local env='';
+
+  if [[ -n "$CONDA_DEFAULT_ENV" ]]; then
```

Comment:
looks good. But probably worth adding a comment explaining why this code exists :)